src/middlewared/middlewared/etc_files/hostname.py

```diff
@@ -0,0 +1,10 @@
+import logging
+
+from middlewared.utils import run
+
+logger = logging.getLogger(__name__)
```

Comment:
Are we logging ?

Reply:
Or maybe we should log if `systemctl` command fails for some reason ?

Reply:
It's just mine (and many other's) standard .py file template